types/src/lib.rs

```diff
@@ -44,6 +48,16 @@ impl PciPath {
 
         Ok(Self(bus, device, function))
     }
+
+    pub fn bus(&self) -> u8 {
+        self.0
+    }
+    pub fn device(&self) -> u8 {
+        self.1
+    }
+    pub fn function(&self) -> u8 {
+        self.2
+    }
```

Comment:
Probably not a hot-path but since this is in a separate crate, worth marking with `#[inline]`?

Reply:
Likely a rounding error when it comes to perf, compared to everything else going on